ClientApp/src/app/airport.service.ts

```diff
@@ -12,7 +11,7 @@ import { Airport } from './Airport';
 
 export class AirportService{
 
-  private airportsUrl = 'api/airports';  // URL to web api
+  private airportsUrl = 'https://localhost:44392/api/airports';  // URL to web api
```

Comment:
You should move base url to some configuration service and inject that in all the places